ironic.conf

```diff
@@ -42,7 +42,9 @@ send_sensor_data = true
 send_sensor_data_interval = 160
 
 [json_rpc]
-auth_strategy = noauth
+# crudini doesn't merge this into ironic.conf as it sees it as redundant (its set globably in DEFAULT)
```

Comment:
This seems like a clear bug in crudini. How can it possibly know that that the option in the DEFAULT group is the fallback for one in another section without an almost-human-level of understanding [the code](https://opendev.org/openstack/ironic/src/branch/master/ironic/common/json_rpc/__init__.py#L20)? This group could be contributed by a completely independent library for all it knows.